src/Components/CategoryCard/CategoryCard.jsx

```diff
@@ -0,0 +1,38 @@
+import React, { useState } from 'react';
+import { useUserData } from '../../Context/UserData-Context';
```

Comment:
suggestion: might do barrell export from context, so that useAuth and useUserData can be imported in a single line